Update/kage_007.txt

```diff
@@ -2018,10 +2018,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 30, "ps3/s08/30/321300092", 256, TRUE);
 	OutputLine(NULL, "「あと、…先ほど情報センターで録音テープを確認したのですが、南井警部の報告がない。",
-			NULL, "\"And... I checked the recordings at the information center earlier, but there were no signs of Chief Inspector Minai.", Line_WaitForInput);
+			NULL, "\"And... I checked the recordings at the information center earlier, but there were no reports from Chief Inspector Minai.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 30, "ps3/s08/30/321300093", 256, TRUE);
 	OutputLine(NULL, "まめに報告する彼女にしては変だと思って調べてみたのですが、",
-			NULL, " That didn't sit well with me, so I did some digging... ", GetGlobalFlag(GLinemodeSp));
+			NULL, " That didn't sit well with me since she frequently reports in, so I did some digging... ", GetGlobalFlag(GLinemodeSp));
```

Comment:
```suggestion
			NULL, " That didn't sit well with me since she usually reports in frequently , so I did some digging... ", GetGlobalFlag(GLinemodeSp));
```